src/main.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
+
+extern crate cli;
+extern crate vmm;
+
+#[cfg(target_arch = "x86_64")]
+use std::convert::TryFrom;
+#[cfg(target_arch = "x86_64")]
+use std::env;
+
+#[cfg(target_arch = "x86_64")]
+use cli::CLI;
+#[cfg(target_arch = "x86_64")]
+use vmm::VMM;
+
+fn main() {
+    #[cfg(target_arch = "x86_64")]
+    {
+        let vmm_config = CLI::launch(
+            env::args()
+                .collect::<Vec<String>>()
+                .iter()
+                .map(|s| s.as_str())
+                .collect(),
+        )
+        .expect("Failed to parse command line options");
```

Comment:
Instead of this expect, can we do something like:
```rust
        if let Ok(vmm_config) = CLI::launch(
            env::args()
                .collect::<Vec<String>>()
                .iter()
                .map(|s| s.as_str())
                .collect(),
        ) {
            let mut vmm = VMM::try_from(vmm_config).expect("Failed to create VMM from configurations");
            vmm.run();
        }
    }
```

The problem with expect is that the program terminates with an ugly panic as follows if you don't pass all the required parameters.
```
thread 'main' panicked at 'Failed to parse command line options: "Invalid command line arguments: \u{1b}[1;31merror:\u{1b}[0m The following required arguments were not provided:\n    \u{1b}[1;31m--kernel <kernel>\u{1b}[0m\n    \u{1b}[1;31m--memory <memory>\u{1b}[0m\n    \u{1b}[1;31m--vcpus <vcpus>\u{1b}[0m\n\nUSAGE:\n    vmm-reference --kernel <kernel> --memory <memory> --vcpus <vcpus>\n\nFor more information try \u{1b}[32m--help\u{1b}[0m\n"', src/main.rs:27:10
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```